src/components/Footer/RenderCompanyInfoSection.jsx

```diff
@@ -5,9 +5,20 @@ import Box from '@mui/material/Box';
 import Link from '@mui/material/Link';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
-import TwitterIcon from '@mui/icons-material/Twitter';
+
 import FacebookIcon from '@mui/icons-material/Facebook';
 import InstagramIcon from '@mui/icons-material/Instagram';
+import SvgIcon from '@mui/material/SvgIcon';
+
+// new Twitter Icon
+const TwitterIcon = (props) => (
+  <SvgIcon {...props} viewBox="0 0 512 512" style={{ width: '1.25em', height: '1.25em' }}>
```

Comment:
With regards to the attributes, think we could use `sx` instead of `style` for MUI.